### ISSUE TITLE
audit-avt1-homepage: video element needs track element

### ISSUE THREAD

**Issue by andreancardona** [carbon-design-system/carbon-website#1416]
## Environment:
> Operating system: Mac
> Browser: Chrome
> AVT :one:
> Page: [Homepage](https://www.carbondesignsystem.com/)

## Detailed Description:
- Media embedded with a <video> element does not have a <track> element to provide text alternatives for audio or visual content

## What to do:
- Use the <track> element with the kind attribute to provide text tracks for the audio (kind="captions") and visual (kind="descriptions") aspects of the video

## Why is this important?
Text tracks contain a textual transcription of either audio (captions) or visual (visual/video description) aspects of video or audio content in a synchronized media. This may include the dialog, sound effects, and relevant musical cues, or information about characters, location, actions and scene changes. Providing text alternatives makes information available that some people would not have access to otherwise.

**Comment by andreancardona** [comment]
@dakahn After bringing this up to the Accessibility team since it seems to be a false positive there has been an issue created to look into the bug: https://github.com/IBMa/equal-access/issues/154

The error is that the checker is expecting video to have captions. However, this video has no audio and so it is a `false positive`

**Comment by andreancardona** [comment]
This issue was merged, closed & fixed.
